.github/workflows/nix.yaml

```
@@ -45,47 +72,44 @@ jobs:
-    #if: |
-    #  (github.event_name != 'pull_request' || github.event.pull_request.merged == true || github.event.action != 'closed')
     outputs:
       docker-image-name: ${{ steps.docker.outputs.image_name }}
+    
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
       - name: Build all flake outputs
+        if: steps.check-conditions.outputs.should_build == 'true'
```

actionlint (1.7.4)

property "check-conditions" is not defined in object type {}

(expression)